sources/packages/backend/apps/api/src/auth/auth.module.ts

```diff
@@ -81,6 +82,10 @@ const jwtModule = JwtModule.register({
       provide: APP_GUARD,
       useClass: SINValidationGuard,
     },
+    {
+      provide: APP_GUARD,
+      useClass: PayloadMaxSizeGuard,
```

Comment:
Maybe I got lost in the middle of some conversation but I was under the impression that we should be having one or another way to have the dynamic payloads restricted.
Is the `PayloadMaxSize` to be used when we have a `dryRun` executed on DTOs with the `@Allow()`?I If that is the case, that was not the intention of the ticket because these types of dynamic-ish DTOs can also receive a `@MaxLength`.
Can we please have a quick dev sync-up on this?
@dheepak-aot thoughts?

Reply:
Looking at `PayloadMaxSize`  I understood that it was for payloads which were absolutely dynamic like student appeals, but looking at education-program controller using it, I didn't get the idea. 
Quick sync up is required here.

Reply:
This `PayloadMaxSizeGuard` is to check the whole payload. There are some requests we use the whole body to save it as a json and there are others that we have a property of the payload object that will be saved as a json. So for those properties we can use the class validator and for the payload we can use the guard. We can discuss that.

Reply:
ok, removed `@PayloadMaxSize` as discussed.